pages/docs/widgets/variants/button.en.mdx

```diff
@@ -53,19 +63,19 @@ These parameters must be supplied via `data-` attributes, otherwise the widget w
 
 ##### `token` string
 
-We use a unique token as an identifier for your organisation. 
+We use a unique token as an identifier for your organisation.
 
 ```html
-<a data-token="Xhpx7QT6xRF9ZHyABrxmm12Q">
+<a data-token="Xhpx7QT6xRF9ZHyABrxmm12Q"></a>
```

Comment:
This change isn't really an improvement and also introduces a difference between this page and the docs of the calculator widget. (This goes for all changes where you've added `</a>`.)